sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/directconnectivity/ReplicatedResourceClient.java

```diff
@@ -157,6 +157,7 @@ public static boolean isGlobalStrongEnabled() {
         int retryTimeout = this.serviceConfigReader.getDefaultConsistencyLevel() == ConsistencyLevel.STRONG ?
                 ReplicatedResourceClient.STRONG_GONE_AND_RETRY_WITH_RETRY_TIMEOUT_SECONDS :
                 ReplicatedResourceClient.GONE_AND_RETRY_WITH_TIMEOUT_IN_SECONDS;
+        logger.info("RetryTimeout: {}", retryTimeout);
```

Comment:
does this need to be info?

Reply:
No - left-over form debugging - can be removed altogether. Thanks for catching!